### ISSUE TITLE
unit-test: Unrecognized extract type adios2

### ISSUE THREAD

**Issue by keyu-Li** [Alpine-DAV/ascent#1002]
Hi,
When I ran the unit test `t_ascent_mpi_adios_extract` in v0.8.0 , I found a problem `Unrecognized extract type adios2`.How should I fix it?
Here is the log:
```
[==========] Running 1 test from 1 test suite.
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from ascent_mpi_runtime
[ RUN      ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime
[----------] Global test environment set-up.
[----------] 1 test from ascent_mpi_runtime
[ RUN      ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from ascent_mpi_runtime
[ RUN      ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime
Rank 1 of 4 reporting
-
  action: "add_extracts"
  extracts:
    e1:
      type: "adios2"
      params:
        engine: "BPFile"
        filename: "/home/lighthouse/code/ascent/src/build/tests/_output/tout_adios_extract.bp"
        actions:
          -
            type: "contour"
            params:
              field: "radial_vert"
              iso_values: 0.3

[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from ascent_mpi_runtime
[ RUN      ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime
Rank 3 of 4 reporting
-
  action: "add_extracts"
Rank 0 of 4 reporting
-
  action: "add_extracts"
  extracts:
    e1:
      type: "adios2"
      params:
        engine: "BPFile"
        filename: "/home/lighthouse/code/ascent/src/build/tests/_output/tout_adios_extract.bp"
        actions:
          -
            type: "contour"
            params:
              field: "radial_vert"
              iso_values: 0.3

Rank 2 of 4 reporting
-
  action: "add_extracts"
  extracts:
    e1:
      type: "adios2"
      params:
        engine: "BPFile"
        filename: "/home/lighthouse/code/ascent/src/build/tests/_output/tout_adios_extract.bp"
        actions:
          -
            type: "contour"
            params:
              field: "radial_vert"
              iso_values: 0.3

  extracts:
    e1:
      type: "adios2"
      params:
        engine: "BPFile"
        filename: "/home/lighthouse/code/ascent/src/build/tests/_output/tout_adios_extract.bp"
        actions:
          -
            type: "contour"
            params:
              field: "radial_vert"
              iso_values: 0.3

[Error] Ascent::execute
file: /home/lighthouse/code/ascent/src/ascent/runtimes/ascent_main_runtime.cpp
line: 812
message:
Unrecognized extract type adios2

[       OK ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime (68 ms)
[----------] 1 test from ascent_mpi_runtime (68 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (68 ms total)
[  PASSED  ] 1 test.
[       OK ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime (76 ms)
[----------] 1 test from ascent_mpi_runtime (77 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (77 ms total)
[  PASSED  ] 1 test.
[       OK ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime (63 ms)
[----------] 1 test from ascent_mpi_runtime (63 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (63 ms total)
[  PASSED  ] 1 test.
[       OK ] ascent_mpi_runtime.test_render_mpi_2d_main_runtime (77 ms)
[----------] 1 test from ascent_mpi_runtime (77 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (77 ms total)
[  PASSED  ] 1 test.
```

**Comment by cyrush** [comment]
@keyu-Li  Can you share how you build Ascent?

If with spack, Adios2 support might not be on by default.
In other cases you will need to provide ADIOS2_DIR, etc when configured for Ascent to leverage Adios2.

**Comment by keyu-Li** [comment]
This is my cmake command.
I find that `t_ascent_mpi_adios_extract` will not  generate if `ADIOS2_DIR` and `FIDES_DIR` are not added.
But now the situation is that `t_ascent_mpi_adios_extract` exists but `adios2` is not found.
```
cmake ..  
-DCONDUIT_DIR=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2 
-DADIOS2_DIR=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/adios2-2.8.1-mvfy7r52eezll4v4jch6cazcjx4hwggj 
-DFIDES_DIR=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/fides-1.1.0-7lonrrd6kpo2afg5fhzgr5qjfsp4ijm6 
-DVTKM_DIR=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/vtk-m-1.7.0-tlpswdqy6bicccprkzob4lr3qa55xceo 
-DVTKH_DIR=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/vtk-h-0.8.1-mgme4outwfvkydmaxm7k6hsjkkshgu74
```
And output is
```
-- BLT Version: 0.3.6
-- CMake Version: 3.23.2
-- CMake Executable: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/cmake-3.23.2-zygscb2cbtm2gbqjgol6fjimj4onrotz/bin/cmake
CMake Deprecation Warning at blt/SetupBLT.cmake:73 (cmake_policy):
  The OLD behavior for policy CMP0076 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
Call Stack (most recent call first):
  cmake/SetupBLT.cmake:35 (include)
  CMakeLists.txt:95 (include)


-- Git Support is ON
-- Git Executable: /usr/bin/git
-- Git Version: 2.25.1
-- MPI Support is ON
-- Enable FindMPI:  ON
-- BLT MPI Compile Flags:  -pthread
-- BLT MPI Include Paths:  /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/include;/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/lib
-- BLT MPI Libraries:      /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/lib/libmpi.so;/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/lib/libmpi_usempif08.so;/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/lib/libmpi_usempi_ignore_tkr.so;/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/lib/libmpi_mpifh.so
-- BLT MPI Link Flags:     SHELL:-Wl,-rpath -Wl,/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/lib -Wl,-rpath -Wl,/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hwloc-2.7.1-5ny4pye6i6rx64nfbfsuomnvgguzopdl/lib -Wl,-rpath -Wl,/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/libevent-2.1.12-r73cwg4vpz3hi4qgxe24qafpwkqabmht/lib -Wl,-rpath -Wl,/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/pmix-4.1.2-fhmi7efccdinegjgrfg53rct5ztsg4c5/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hwloc-2.7.1-5ny4pye6i6rx64nfbfsuomnvgguzopdl/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/libevent-2.1.12-r73cwg4vpz3hi4qgxe24qafpwkqabmht/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/pmix-4.1.2-fhmi7efccdinegjgrfg53rct5ztsg4c5/lib -pthread
-- MPI Executable:       /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openmpi-4.1.4-bnd2weialepmcqidcjdubopwxliynt5e/bin/mpiexec
-- MPI Num Proc Flag:    -n
-- MPI Command Append:
-- Using MPI Fortran header: mpif.h
-- OpenMP Support is OFF
-- CUDA Support is OFF
-- HIP Support is OFF
-- HCC Support is OFF
-- Sphinx support is ON
-- Failed to locate Sphinx executable (missing: SPHINX_EXECUTABLE)
-- Valgrind support is ON
-- AStyle support is ON
-- Failed to locate AStyle executable (missing: ASTYLE_EXECUTABLE)
-- ClangFormat support is ON
-- Failed to locate ClangFormat executable (missing: CLANGFORMAT_EXECUTABLE)
-- Uncrustify support is ON
-- Failed to locate Uncrustify executable (missing: UNCRUSTIFY_EXECUTABLE)
-- Cppcheck support is ON
-- Failed to locate Cppcheck executable (missing: CPPCHECK_EXECUTABLE)
-- ClangQuery support is ON
-- Failed to locate ClangQuery executable (missing: CLANGQUERY_EXECUTABLE)
-- C Compiler family is GNU
-- Fortran Compiler family is GNU
-- Adding optional BLT definitions and compiler flags
-- Standard C++11 selected
-- Fortran support enabled.
-- CMAKE_C_FLAGS flags are:
-- CMAKE_CXX_FLAGS flags are:
-- CMAKE_Fortran_FLAGS flags are:
-- CMAKE_EXE_LINKER_FLAGS flags are:
-- Google Test Support is ON
-- Google Mock Support is OFF
CMake Deprecation Warning at blt/thirdparty_builtin/googletest-master-2020-01-07/googletest/CMakeLists.txt:56 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Fruit Support is ON
-- CMake build tool name: /usr/bin/make
-- Building shared libraries (BUILD_SHARED_LIBS == ON)
-- Building without coverage flags (ENABLE_COVERAGE == OFF)
-- git executable: /usr/bin/git
-- git SHA1: 08504374908518e013d7fe8d8882cfb1c2378e3b
-- git SHA1-abbrev: 08504
-- git tag: v0.8.0
-- Fortran Compiler: /usr/bin/gfortran
-- PYTHON_EXECUTABLE /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/bin/python
-- PYTHON_CONFIG_VERSION 3.7
-- PYTHON_INCLUDE_DIR /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/include/python3.7m
-- PYTHON_SITE_PACKAGES_DIR /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/lib/python3.7/site-packages
-- PYTHON_LDSHARED_FLAGS /usr/bin/gcc -pthread -shared -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/bzip2-1.0.8-oobld7xhovhcgx6nrwujeyocl62j437y/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/expat-2.4.8-uys5fepttcbhjdxbbcj5qf2opnzbptus/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/gdbm-1.19-nn6ceu3lbvv72wymfgpudiv3a6czjvy2/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/gettext-0.21-nqglzm3cb4xhw356amxp6ifmtb62yncb/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/libffi-3.4.2-xtpnvsq4rb265rvthajmlachqzbq427x/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/ncurses-6.2-3zcrtn7jf6yuqs5sdb2m47jbdtjzzusi/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/openssl-1.1.1o-hkquni7fdoqsiemetf5ucqwuzxto4xkm/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/readline-8.1-livagh6uupewhznnxdelkd4hbjki6cp2/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/sqlite-3.38.5-7xp7mzkx25t2uivwiiycs3dssz2uaksw/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/util-linux-uuid-2.37.4-wj3zixhhgpmv66llhx3f5ty4vtyfaz2f/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/xz-5.2.5-7h7t6ejizlndpwj65m35zgqovxvnadwa/lib -L/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/zlib-1.2.12-ckadtmkfuxuyizgp5djpvv7felahwrcf/lib
-- PYTHON_USE_UNDEFINED_DYNAMIC_LOOKUP_FLAG is OFF
-- PYTHON_CONFIG_LIBDIR:     /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/lib
-- PYTHON_CONFIG_LIBPL:      /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/lib/python3.7/config-3.7m-x86_64-linux-gnu
-- PYTHON_CONFIG_LDLIBRARY:  libpython3.7m.so
-- PYTHON_CONFIG_LIBRARY:    libpython3.7m.a
-- Checking for python library at: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/lib/libpython3.7m.so
-- {PythonLibs from PythonInterp} using: PYTHON_LIBRARY=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/lib/libpython3.7m.so
-- Using Python Include: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/include/python3.7m
-- Looking for Conduit using CONDUIT_DIR = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2
-- Conduit was built with HDF5 Support
-- Looking for HDF5 at: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7
-- Found HDF5: hdf5-shared (found version "1.8.22")
-- HDF5_DIR_REAL=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7
-- Checking that found HDF5_INCLUDE_DIRS are in HDF5_DIR
-- HDF5_INCLUDE_DIRS=/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7/include
--  /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7/include includes HDF5_DIR (/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7)
--  /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7/include includes HDF5_REAL_DIR ()
-- HDF5 is parallel:  ON
-- Found Conduit: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2 (found version 0.8.3)
-- CONDUIT_VERSION             = 0.8.3
-- CONDUIT_INSTALL_PREFIX      = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2
-- CONDUIT_IMPORT_ROOT         = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2
-- CONDUIT_USE_CXX11           = TRUE
-- CONDUIT_USE_FMT             = TRUE
-- CONDUIT_INCLUDE_DIRS        = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2/include/conduit
-- CONDUIT_FORTRAN_ENABLED     = TRUE
-- CONDUIT_PYTHON_ENABLED      = TRUE
-- CONDUIT_PYTHON_EXECUTABLE   = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/python-3.7.13-e2ba6owds753h2ym53ra5kcjvgaavbhp/bin/python3.7
-- CONDUIT_PYTHON_MODULE_DIR   = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2/lib/python3.7/site-packages
-- Conduit Relay features:
--  CONDUIT_RELAY_WEBSERVER_ENABLED = TRUE
--  CONDUIT_RELAY_HDF5_ENABLED      = TRUE
--  CONDUIT_HDF5_DIR                = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/hdf5-1.8.22-ljwbam4ehlf4aarwvyttw2a327coyka7
--  CONDUIT_RELAY_ADIOS_ENABLED     = FALSE
--  CONDUIT_ADIOS_DIR               =
--  CONDUIT_RELAY_SILO_ENABLED      = FALSE
--  CONDUIT_SILO_DIR                =
--  CONDUIT_RELAY_MPI_ENABLED       = TRUE
-- Conduit imported targets: conduit::conduit conduit::conduit_python conduit::conduit_mpi
-- FOUND conduit fortran module at: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2/include/conduit/conduit.mod
-- CONDUIT_RELAY_WEBSERVER_ENABLED = TRUE
-- PYTHON found!! /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2
-- FOUND conduit python module at: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-x5ognuohvboqv6auqnxkd3keuhz3qevw/lib/python3.7/site-packages/conduit
-- FOUND conduit python include dir: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-x5ognuohvboqv6auqnxkd3keuhz3qevw/lib/python3.7/site-packages/conduit
-- FOUND Conduit at /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2
-- CONDUIT_INCLUDE_DIRS = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-depzz7pom6uyi5icfwyiix56zg5bmjx2/include/conduit;/home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/conduit-0.8.3-x5ognuohvboqv6auqnxkd3keuhz3qevw/lib/python3.7/site-packages/conduit
-- Looking for VTKm using VTKM_DIR = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/vtk-m-1.7.0-tlpswdqy6bicccprkzob4lr3qa55xceo
-- Looking for VTKh using VTKH_DIR = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/vtk-h-0.8.1-mgme4outwfvkydmaxm7k6hsjkkshgu74
-- VTKh using provided path  VTKM_DIR: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/vtk-m-1.7.0-tlpswdqy6bicccprkzob4lr3qa55xceo
-- Found VTKh include dirs: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/vtk-h-0.8.1-mgme4outwfvkydmaxm7k6hsjkkshgu74/include
-- Looking for ADIOS2 using ADIOS2_DIR = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/adios2-2.8.1-mvfy7r52eezll4v4jch6cazcjx4hwggj
-- Found ADIOS2: /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/adios2-2.8.1-mvfy7r52eezll4v4jch6cazcjx4hwggj/lib/cmake/adios2/adios2-config.cmake (found version "2.8.1") found components: C CXX Fortran MPI
-- FOUND ADIOS2 at /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/adios2-2.8.1-mvfy7r52eezll4v4jch6cazcjx4hwggj/lib/cmake/adios2
-- Looking for FIDES using FIDES_DIR = /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/fides-1.1.0-7lonrrd6kpo2afg5fhzgr5qjfsp4ijm6
-- Found Fides at /home/lighthouse/spack/opt/spack/linux-ubuntu20.04-x86_64_v3/gcc-9.4.0/fides-1.1.0-7lonrrd6kpo2afg5fhzgr5qjfsp4ijm6
-- Tests are enabled (ENABLE_TESTS=ON)
-- Configuring hybrid python module: flow_python
-- Configuring python distutils setup: flow_python_py_setup
-- Configuring python module: flow_python
-- flow_python build location: /home/lighthouse/code/ascent/src/build/python-modules/flow
-- Configuring python distutils setup: ascent_python_py
-- Configuring python module: ascent_python
-- ascent_python build location: /home/lighthouse/code/ascent/src/build/python-modules/ascent
-- Configuring python module: ascent_mpi_python
-- ascent_mpi_python build location: /home/lighthouse/code/ascent/src/build/python-modules/ascent/mpi
-- Adding vtk-h lib unit tests
--  [*] Adding Unit Test: t_vtkh_smoke
-- Adding flow lib unit tests
--  [*] Adding Unit Test: t_flow_data
--  [*] Adding Unit Test: t_flow_timer
--  [*] Adding Unit Test: t_flow_registry
--  [*] Adding Unit Test: t_flow_workspace
--  [*] Adding Unit Test: t_flow_workspace_adv_manage
--  [*] Adding Unit Test: t_flow_python_interpreter
--  [*] Adding Unit Test: t_flow_python_script_filter
-- Adding flow lib python interface unit tests
--  [*] Adding Python-based Unit Test: t_python_flow_smoke
--  [*] Adding Python-based Unit Test: t_python_flow_inception
-- Adding ascent lib unit tests
--  [*] Adding Unit Test: t_ascent_smoke
--  [*] Adding Unit Test: t_ascent_runtime_options
--  [*] Adding Unit Test: t_ascent_utils
--  [*] Adding Unit Test: t_ascent_memory
--  [*] Adding Unit Test: t_ascent_derived
--  [*] Adding Unit Test: t_ascent_empty_runtime
--  [*] Adding Unit Test: t_ascent_expressions
--  [*] Adding Unit Test: t_ascent_cinema_a
--  [*] Adding Unit Test: t_ascent_render_2d
--  [*] Adding Unit Test: t_ascent_render_3d
--  [*] Adding Unit Test: t_ascent_render_2d_poly
--  [*] Adding Unit Test: t_ascent_render_3d_poly
--  [*] Adding Unit Test: t_ascent_web_flow
--  [*] Adding Unit Test: t_ascent_web_main
--  [*] Adding Unit Test: t_ascent_clip
--  [*] Adding Unit Test: t_ascent_scalar_renderer
--  [*] Adding Unit Test: t_ascent_multi_topo
--  [*] Adding Unit Test: t_ascent_gradient
--  [*] Adding Unit Test: t_ascent_qcriterion
--  [*] Adding Unit Test: t_ascent_divergence
--  [*] Adding Unit Test: t_ascent_vorticity
--  [*] Adding Unit Test: t_ascent_relay
--  [*] Adding Unit Test: t_ascent_clip_with_field
--  [*] Adding Unit Test: t_ascent_contour
--  [*] Adding Unit Test: t_ascent_iso_volume
--  [*] Adding Unit Test: t_ascent_image_compare
--  [*] Adding Unit Test: t_ascent_pipelines_to_pipelines
--  [*] Adding Unit Test: t_ascent_threshold
--  [*] Adding Unit Test: t_ascent_slice
--  [*] Adding Unit Test: t_ascent_scale
--  [*] Adding Unit Test: t_ascent_particle_advection
--  [*] Adding Unit Test: t_ascent_patch_amr
--  [*] Adding Unit Test: t_ascent_vector_ops
--  [*] Adding Unit Test: t_ascent_flow_runtime
--  [*] Adding Unit Test: t_ascent_recenter
--  [*] Adding Unit Test: t_ascent_rover
--  [*] Adding Unit Test: t_ascent_lagrangian
--  [*] Adding Unit Test: t_ascent_log
--  [*] Adding Unit Test: t_ascent_amr
--  [*] Adding Unit Test: t_ascent_queries
--  [*] Adding Unit Test: t_ascent_failed_pipeline
--  [*] Adding Unit Test: t_ascent_triggers
--  [*] Adding Unit Test: t_ascent_hola
--  [*] Adding Unit Test: t_ascent_sampling
--  [*] Adding Unit Test: t_ascent_ascent_runtime
--  [*] Adding Unit Test: t_ascent_python_extract
--  [*] Adding Unit Test: t_ascent_vtkh_data_adapter
-- MPI enabled: Adding related unit tests
--  [*] Adding Unit Test: t_ascent_mpi_smoke
--  [*] Adding Unit Test: t_ascent_mpi_empty_runtime
--  [*] Adding Unit Test: t_ascent_mpi_derived
--  [*] Adding Unit Test: t_ascent_mpi_expressions
--  [*] Adding Unit Test: t_ascent_mpi_render_2d
--  [*] Adding Unit Test: t_ascent_mpi_render_3d
--  [*] Adding Unit Test: t_ascent_mpi_statistics
--  [*] Adding Unit Test: t_ascent_mpi_multi_topo
--  [*] Adding Unit Test: t_ascent_mpi_slice
--  [*] Adding Unit Test: t_ascent_mpi_ascent_runtime
--  [*] Adding Unit Test: t_ascent_mpi_relay_extract
--  [*] Adding Unit Test: t_ascent_mpi_adios_extract
--  [*] Adding Unit Test: t_ascent_mpi_python_extract
--  [*] Adding Unit Test: t_ascent_hola_mpi
-- Adding ascent lib python interface unit tests
--  [*] Adding Python-based Unit Test: t_python_ascent_smoke
--  [*] Adding Python-based Unit Test: t_python_ascent_render
--  [*] Adding Python-based Unit Test: t_python_ascent_flow_runtime
--  [*] Adding Python-based MPI Unit Test: t_python_ascent_mpi_render
-- Adding ascent lib fortran interface unit tests
--  [*] Adding Fortran Unit Test: t_f_ascent_smoke
--  [*] Adding Fortran Unit Test: t_f_ascent_render_2d
-- Adding performance tests
--  [*] Adding performance test: t_perf_clover
-- Configuring done
-- Generating done
-- Build files have been written to: /home/lighthouse/code/ascent/src/build
```

**Comment by cyrush** [comment]
thanks for the extra details.

From your report -- I found bugs in the logic that enables our adios2 extract.

They will be resolved via:
https://github.com/Alpine-DAV/ascent/pull/1010

and the fixes will be in Ascent 0.9.0.

**Comment by cyrush** [comment]
resolved in #1010
